frontend/src/templates/Field/Table/ColumnCell.tsx

```diff
@@ -82,12 +84,52 @@ const DropdownColumnCell = ({
   inputName,
   colorTheme,
 }: FieldColumnCellProps<DropdownColumnBase>) => {
+  const { i18n } = useTranslation()
   const { control } = useFormContext<TableFieldInputs>()
   const rules = useMemo(
     () => createDropdownValidationRules(schema, disableRequiredValidation),
     [schema, disableRequiredValidation],
   )
 
+  const selectedLanguage = i18n.language as Language
+  const fieldOptions: ComboboxItem[] = useMemo(() => {
+    const defaultEnglishFieldOptions = schema.fieldOptions ?? []
+    const fieldOptionsTranslations = schema?.fieldOptionsTranslations ?? []
+    const translationIdx = fieldOptionsTranslations.findIndex(
+      (translation) => translation.language === selectedLanguage,
+    )
+
+    // Check if translations for field options exist and whether
+    // each field option has its own respective translation. If not
+    // render the default field options in English.
+    if (
+      translationIdx !== -1 &&
+      fieldOptionsTranslations[translationIdx].translation.length ===
+        defaultEnglishFieldOptions.length
+    ) {
+      const translatedFieldOptions =
+        fieldOptionsTranslations[translationIdx].translation
```

Comment:
Same question as in `frontend/src/templates/Field/Dropdown/DropdownField.tsx` regarding the utilisation of multi-lang utils. Was it due to a need for custom extractor?